conf/evolutions/default/85.sql

```diff
@@ -0,0 +1,12 @@
+# --- !Ups
+
+INSERT INTO `vitals` (`id`, `name`, `data_type`, `unit_of_measurement`)
+  VALUES ('11', 'weeksPregnant', 'float', 'weeks');
```

Comment:
Great job making this evolution! One small thing though - We don't really want to manually insert the id. MySQL will auto increment the Id for us. Remove the id portion of the insert and the rest should be good.

Even though assigning the Id of 11 should be safe, we still want this evolution to work if for some reason id 11 already exists in someone's femr install. Some early evolutions do not follow this though.